src/bosh-director/spec/unit/deployment_plan/manual_network_subnet_spec.rb

```diff
@@ -105,16 +105,29 @@ def create_dynamic_reservation(ip)
         [],
       )
 
-      expect(subnet.range.ip).to eq('fdab:d85c:118d:8a46:0000:0000:0000:0000')
-      subnet.range.ip.size == 2**64
-      expect(subnet.netmask).to eq('ffff:ffff:ffff:ffff:0000:0000:0000:0000')
-      expect(subnet.gateway).to eq('fdab:d85c:118d:8a46:0000:0000:0000:0001')
+      expect(subnet.range.network.to_s).to eq('fdab:d85c:118d:8a46::')
+      # subnet.range.ip.size == 2**64  # TODO NETADDR: bug? / what was tested here?
```

Comment:
That CIDR Prefix is parsed into a range size. Should probably be updated to `network.prefix_len`